src/test/protocol/Memo_test.cpp

```diff
@@ -72,23 +72,23 @@ class Memo_test : public beast::unit_test::suite
             auto& m = mi[sfCreatedNode.jsonName];  // CreatedNode in Memos
             m[sfMemoData.jsonName] = "3030303030";
 
-            env(memoNonMemo, ter(temINVALID));
+            env(memoNonMemo, ter(telENV_RPC_FAILED));
         }
         {
             // Put an invalid field in a Memo object.
             JTx memoExtra = makeJtxWithMemo();
             memoExtra
                 .jv[sfMemos.jsonName][0u][sfMemo.jsonName][sfFlags.jsonName] =
                 13;
-            env(memoExtra, ter(temINVALID));
+            env(memoExtra, ter(telENV_RPC_FAILED));
```

Comment:
To emphasize my point: this test case underscores invalid fields in a memo object. It is more appropriate to return a `temINVALID` code, instead of the current change. That would help localize `telENV_RPC_FAILED` to a few places in the code base.

Reply:
My explanation above covers this test case as well.